### DISCUSSION TITLE
Change SecurityOptions with colima start --edit

### DISCUSSION THREAD

**Discussion by gibbardsteve** [abiosoft/colima#1019] [General]
Hi,

TLDR; How do I disable apparmor in Security Options using colima start --edit (I see it enabled when running docker info)?

I am experiencing a failing container when using colima, but when I run with docker desktop the container works fine. I think I have narrowed it down to a permissions issue when mounting directories. What I would like to do in Colima is set the docker config to match the working docker desktop scenario to see if that is the issue.

running _docker info_ in working scenario (docker desktop):
 Security Options:
  seccomp
   Profile: unconfined
  cgroupns
  
running _docker info_ in colima:
Security Options:
  apparmor
  seccomp
   Profile: unconfined
  cgroupns  

How can I edit the colima config template to indicate apparmor is disabled?

My current config in the template file (when I run colima start --edit) is:

docker:
  seccomp-profile: unconfined

I have tried apparmor: false but this doesn't work.

**Comment by Stokestack** [comment]
What errors are you getting? I may be having the same problem. I'm trying to run Supabase, which I was able to do with Docker Desktop running. But with Colima I get tons of

`supabase-db                     | chown: changing ownership of '/var/lib/postgresql/data': Permission denied`

My Docker info shows:
```
Security Options:
  apparmor
  seccomp
   Profile: builtin
  cgroupns
```

**Comment by hwo411** [comment]
Would be great to have it. I have another issue with AppArmor.

When you use devspace, it runs docker registry inside the k8s which fails because AppArmor turned on, and the instructions to disable it per user don't help. I had to turn AppArmor off manually, but it'd be great to be able to do this automatically through config.
